client/src/pages/private/SiteTable.js

```diff
@@ -286,17 +302,39 @@ export default ({ sites, viewOnly }) => {
         </Grid>
 
         <CheckPermissions roles={roles} permittedRoles={['ministry_of_health']}>
-          <Grid item xs={2} />
-          <Grid className={classes.rootItem} item xs={2}>
-            <Button
-              onClick={() => {
-                setActiveModalForm('new-site');
-              }}
-              size='medium'
-              text='Create Site'
-              startIcon={<AddCircleOutlineIcon />}
-            />
-          </Grid>
+          <Grid item xs={8} />
+          <ClickAwayListener onClickAway={() => setActionMenuOpen(false)}>
+            <Grid className={classes.rootItem} item xs={2}>
+              <Button
```

Comment:
What are the chances we have this kind of action menu elsewhere or that we will have more in the future? This seems like a good candidate for a reusable component if it doesn't already exist

Also was there a reason not to use the [MUI menu component](https://v4.mui.com/components/menus/#menus)?